core/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/DataPlaneFrameworkExtension.java

```diff
@@ -77,7 +78,8 @@ public String name() {
 
     @Override
     public void initialize(ServiceExtensionContext context) {
-        var pipelineService = new PipelineServiceImpl();
+        var monitor = context.getMonitor();
```

Comment:
Can you please change all of the log messages below from info to debug and consolidate them into one message?

Reply:
> Can you please change all of the log messages below from info to debug and consolidate them into one message?

I think those one-time message on staring up are worth for INFO level. It is quite useful to show setup in the startup message for finding issues like loading unintentional configuration files on operation.

Reply:
I updated the log level and consolidated the message.